### PR TITLE
add chinese translation of ianvs/sendas with help of chatgpt

### PR DESCRIPTION
**Pull request by magichan**
* **Please check if the PR fulfills these requirements**

- [x] The commit message follows our guidelines
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features)

**Which issue(s) this PR fixes**:

Fixes https://github.com/kubeedge/website/issues/619

* **What kind of change does this PR introduce?** (Bug fix, feature, docs update, ...)

bug fix

* **What is the current behavior?** (You can also link to an open issue here)

Display English documents in Chinese mode

* **What is the new behavior (if this is a feature change)?**

translate from English to Chinese

* **Does this PR introduce a breaking change?** (What changes might users need to make in their application due to this PR?)

no

* **Other information**:

### CODE REVIEW

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md [hunk 1]
```diff
@@ -3,67 +3,68 @@ title: Sedna
 sidebar_position: 1
 ---
 
-## What is Sedna?
+## Senda 是什么？
 
-Sedna is an edge-cloud synergy AI project incubated in KubeEdge SIG AI. Benefiting from the edge-cloud synergy capabilities provided by KubeEdge, Sedna can implement across edge-cloud collaborative training and collaborative inference capabilities, such as joint inference, incremental learning, federated learning, and lifelong learning. Sedna supports popular AI frameworks, such as TensorFlow, Pytorch, PaddlePaddle, MindSpore.
+Senda 是由 KubeEdge SIG AI 孵化的边缘云协同 AI 项目。受益于 KubeEdge 提供的边缘云协同能力，Sedna 可以实现跨越云和边的协同训练和协同推理能力，如联合推理、增量学习、联邦学习和终身学习。Sedna 支持流行的 AI 框架，如 TensorFlow、Pytorch、PaddlePaddle、MindSpore。
```
Comment (Shelley-BaoYue) [inline i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md:8]:
```suggestion
Senda 是由 KubeEdge SIG AI 孵化的边云协同 AI 项目。受益于 KubeEdge 提供的边云协同能力，Sedna 可以实现跨越云和边的协同训练和协同推理能力，如联合推理、增量学习、联邦学习和终身学习。Sedna 支持流行的 AI 框架，如 TensorFlow、Pytorch、PaddlePaddle、MindSpore。
```

**Comment by Shelley-BaoYue** [comment]
cc @MooreZheng @hsj576

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md (hunk 1 above)
Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md:8]: Thx，

If `边云协同` is a more preferred translation than `云边协同`, then I will revise the whole thing in the evening.

Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md]: change all `边缘云协同` and `云边协同` to `边云协同`

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md [hunk 2]
```diff
@@ -3,67 +3,68 @@ title: Sedna
...
-### Component
-Sedna consists of the following components：
+### 组件
+Sedna 包括以下组件：
 
-#### GlobalManager
-* Unified edge-cloud synergy AI task management
-* Cross edge-cloud synergy management and collaboration
-* Central Configuration Management
+#### 全局管理器
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md:48]: The component name does not need to be translated, just use `GlobalManager` directly

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md [hunk 3]
```diff
@@ -3,67 +3,68 @@ title: Sedna
...
+* 统一的边云协同 AI 任务管理
+* 跨云边的协同管理和协作
+* 中央配置管理
 
-#### LocalController
-* Local process control of edge-cloud synergy AI tasks
-* Local general management: model, dataset, and status synchronization
+#### 本地控制器
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md:53]: ditto

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md [hunk 4]
```diff
@@ -3,67 +3,68 @@ title: Sedna
...
+* 边云协同 AI 任务的本地流程控制
+* 本地通用管理：模型、数据集和状态同步
 
 
-#### Worker
-* Do inference or training, based on existing ML framework.
-* Launch on demand, imagine they are docker containers.
-* Different workers for different features.
-* Could run on edge or cloud.
+#### 工作节点
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md:58]: ditto

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md [hunk 5]
```diff
@@ -3,67 +3,68 @@ title: Sedna
...
+* 根据现有的 ML 框架进行推理或训练。
+* 按需启动，可以想象它们是 docker 容器。
+* 不同的工作节点用于不同的特性。
+* 可以在边缘或云上运行。
 
 
-#### Lib
-* Expose the Edge AI features to applications, i.e. training or inference programs.
+#### 库
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md:65]: ditto

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md [hunk 6]
```diff
@@ -5,59 +5,58 @@ sidebar_position: 2
...
-  - Tools to manage test environment. For example, it would be necessary to support the CRUD (Create, Read, Update and Delete) actions in test environments. Elements of such test environments include algorithm-wise and system-wise configuration.
-  - Tools to control test cases. Typical examples include paradigm templates, simulation tools, and hyper-parameter-based assistant tools.
-  - Tools to manage benchmark presentation, e.g., leaderboard and test report generation.
-- Cooperation with other organizations or communities, e.g., in KubeEdge SIG AI, to establish comprehensive benchmarks and developed related applications, which can include but are not limited to
-  - Dataset collection, re-organization, and publication
-  - Formalized specifications, e.g., standards
-  - Holding competitions or coding events, e.g., open source promotion plan
-  - Maintaining solution leaderboards or certifications for commercial usage
+Ianvs 的职责范围包括：
+- 基于典型的分布式协同 AI 范式和应用程序，提供跨设备、且覆盖边缘节点和云节点的端到端基准工具包。
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md:14]:
```suggestion
- 基于典型的分布式协同 AI 范式和应用程序，提供跨设备且覆盖边缘节点和云节点的端到端基准工具包。
```

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md [hunk 7]
```diff
@@ -5,59 +5,58 @@ sidebar_position: 2
...
-  - Generation Assistant: assist users to generate test cases based on certain rules or constraints, e.g., the range of parameters
-  - Simulation Controller: control the simulation process of edge-cloud synergy AI, including the instance generation and vanishment of simulation containers
-- Story Manager: the output management and presentation of the test case, e.g., leaderboards
+## 架构
+下图显示了架构和相关概念。Ianvs 被设计为在单个节点内运行。关键组件包括
+- 测试环境管理器(Test Environment Manager)：用于全局使用的测试环境的创建、读取、更新和删除 (CRUD)
+- 测试用例控制器(Test Case Controller)：控制测试用例的运行时行为，如实例生成和删除
+  - 生成助手(Generation Assistant)：帮助用户根据某些规则或约束生成测试用例，例如参数范围
+  - 模拟控制器(Simulation Controller)：控制云边协同 AI 工具的模拟过程，包括模拟容器的实例生成和消失
+- 故事管理器(Story Manager)：测试用例的输出管理和展示，例如排行榜
```
Comment (wbc6080) [inline i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md:31]: Component names do not need to be translated

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md (hunk 2 above)
Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md]: Done

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md (hunk 3 above)
Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md]: Done

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md (hunk 4 above)
Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md]: Done

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md (hunk 5 above)
Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/sedna.md]: Done

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md (hunk 6 above)
Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md]: Done

File: i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md (hunk 7 above)
Comment (magichan) [reply i18n/zh/docusaurus-plugin-content-docs/current/concept/ai/ianvs.md]: keep english name of components and remove chinese name.

done

**Review by wbc6080 — COMMENTED** [review]
/lgtm

**Review by fisherxu — COMMENTED** [review]
/approve

**Pull request merged**